blog/2020-05-13-native-server-announce.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog
+title: Infinispan Native Server Image
+date: '2020-05-132T12:00:00.000-00:00'
+author: Ryan Emerson
+tags: [ "docker", "native", "quarkus" ]
+---
+Starting with Infinispan 11, it's now possible to create a natively compiled version of the Infinispan server.
+
+== TL;DR
+We have a new image that contains a natively compiled Infinispan server and has a footprint of only 281MB. Try it now:
+
+----
+docker run -p 11222:11222 quay.io/infinispan/server-native:11.0
+----
+
+== Infinispan Quarkus Extensions
+
+https://quarkus.io/[Quarkus] provides built in support for generating native executables, providing several
+abstractions to improve the development experience of creating native binaries. Building upon the
+https://infinispan.org/blog/2019/11/11/serverng/[new server], the Infinispan team have created a
+https://github.com/infinispan/infinispan-quarkus[Quarkus extension] for both embedded and server use-cases.
+These extensions allow a native binary version of the server to be compiled and ran by simply executing:
+
+----
+mvn clean install -Dnative
+./server-runner/target/infinispan-quarkus-server-runner-11.0.0.Dev05-runner
+    -Dquarkus.infinispan-server.config-file=infinispan.xml \
+    -Dquarkus.infinispan-server.config-path=server/conf \
+    -Dquarkus.infinispan-server.data-path=data \
+    -Dquarkus.infinispan-server.server-path=/opt/infinispan &
+----
+
+== Native Server Image
+For many developers compiling your own Infinispan native binary manually is not desirable, therefore we
+provide the `infinispan/server-native` image that utilise a native server binary. The advantage of this over our JVM
+based `infinispan/server` image is that we can no provide a much smaller image, 281 vs 571 MB, as we no longer need to
+include an openjdk JVM in the image.
```

Comment:
Again, some RSS info too

Reply:
And startup time :)

Reply:
Startup time in the image isn't much different, because we're stuck waiting for JGroups to start

Reply:
That is because we have bad jgroups timeouts. And try a second node, you will change your mind.